modules/investigating-kernel-crashes.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operating-system-issues.adoc
+
+[id="investigating-kernel-crashes"]
+= Investigating kernel crashes
+
+== Enabling `kdump`
+
+The `kdump` service, included in `kexec-tools`, provides a crash-dumping mechanism. You can use this service to save the contents of the system’s memory for later analysis.
+
+{op-system} ships with `kexec-tools`, but manual configuration is required to enable `kdump`.
+
+.Procedure
+
+The following steps are needed to enable `kdump` on {op-system}.
+
+.  Memory must be reserved for the crash kernel during booting of the first kernel. Provide kernel arguments by entering the following command:
++
+[source, terminal]
+----
+sudo rpm-ostree kargs --append='crashkernel=256M'
+----
+
+.  By default, the path in which the vmcore will be saved is `/var/crash`. It is also possible to write the dump over the network or to some other location on the local system by editing `/etc/kdump.conf`. For example, assuming `/var/usrlocal/cores` exists, enter the following command to edit `/etc/kdump.conf` to save the vmcore to `/var/usrlocal/cores`:
```

Comment:
Tiny nit to single-space after `.` for list items:
s/.  By default/. By default

Reply:
Ah, I wonder where I got the idea to put two spaces.